content/2017/04-everything-not-saved-will-be-lost.rst

```diff
@@ -4,12 +4,25 @@
 :date: 2017-03-13
 :description: Everything not saved will be lost. (Nintendo Quit Screen message)
 
-TODO: intro text
+Nix, NixOS and nixpkgs are all moving ahead at good speed. The `17.03 milestone`_ is more than 50% complete, and we merged 144 pull requests for nixpkgs just in the last seven days.
+
+.. _`17.03 milestone`: https://github.com/NixOS/nixpkgs/milestone/10
+
 
 News
 ====
 
-TODO:
+- We now allow packages to be marked insecure, and you have to opt-in to install insecure packages similarly to how you opt in to non-free software.  The option is called ``permittedInsecurePackages``. See the `updated docs`_ for more.
+
+- Robin finished the `systemd 232 update`_ which opens up several new `opportunities for hardening`_. It will also allow us to move away from hard-coded user IDs for less important services. Just use ``DynamicUser = true``.
```

Comment:
@globin dunno if we mentioned it already, if not we should keep it IMO (at least on my radar it did not yet appear :D )